vispy/app/backends/_wx.py

```diff
@@ -218,13 +218,18 @@ def __init__(self, *args, **kwargs):
         else:
             self._gl_context = p.context.shared.ref._gl_context
 
+        if p.position == None:
+            pos = wx.DefaultPosition
```

Comment:
Is this available in all modern versions of wx? Or only the newest one?

Reply:
```grep``` show that ```wx.DefaultPosition``` used in https://github.com/wxWidgets/wxPython

Reply:
https://github.com/wxWidgets/wxPython/blob/8960e7ee0b6d11c5ff02f5e89665f0bdf50b7c62/src/gtk/_core.py#L2223